### ISSUE TITLE
 rdoc/markup/table.rb is missing in v6.3.1

### ISSUE THREAD

**Issue by ainame** [ruby/rdoc#808]
Thanks to a `fastlane` gem user's report, I noticed that `rdoc/markup/table.rb` is missing in `rdoc` v6.3.1, which causes installation failures for `gem install fastlane`.
https://github.com/fastlane/fastlane/issues/18642

As far as I investigated, https://my.diffend.io/gems/rdoc/6.3.0/6.3.1/page/2#d2h-013500-56 this line that points to `rdoc/markup/table` was added to 6.3.1, however, this version doesn't contain the file pointed (it exists on `master` branch) and then it causes an error when installing a gem like `fastlane`.  (I confirmed that by unarchiving distributed `.gem` file.)

Given 6.3.1 is meant to a security fix, I appreciate it if we can get another release to fix this installation issue so that affected users can protect themselves 🙇 

## Steps to reproduce 

```
$ rbenv local 2.7.2 # I tested it with 2.7 but perhaps it can happen on other versions as well
$ gem install rdoc # This should install 6.3.1 now
$ gem install fastlane #=> You get error. The details are attached below.  
```

<details>

```
satoshinamai@macbook fastlane % gem install fastlane-2.182.0.gem
Successfully installed fastlane-2.182.0
Parsing documentation for fastlane-2.182.0
Installing ri documentation for fastlane-2.182.0
Traceback (most recent call last):
	45: from /Users/satoshinamai/.rbenv/versions/2.7.2/bin/gem:21:in `<main>'
	44: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/gem_runner.rb:59:in `run'
	43: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/command_manager.rb:148:in `run'
	42: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/command_manager.rb:178:in `process_args'
	41: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/command.rb:325:in `invoke_with_build_args'
	40: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/commands/install_command.rb:165:in `execute'
	39: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/commands/install_command.rb:217:in `install_gems'
	38: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/commands/install_command.rb:217:in `each'
	37: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/commands/install_command.rb:224:in `block in install_gems'
	36: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/commands/install_command.rb:208:in `install_gem'
	35: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/request_set.rb:221:in `install'
	34: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/request_set.rb:322:in `install_hooks'
	33: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/request_set.rb:322:in `each'
	32: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/request_set.rb:323:in `block in install_hooks'
	31: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/rubygems_hook.rb:55:in `generation_hook'
	30: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/rubygems_hook.rb:55:in `each'
	29: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/rubygems_hook.rb:56:in `block in generation_hook'
	28: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/rubygems_hook.rb:193:in `generate'
	27: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/rubygems_hook.rb:134:in `document'
	26: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/rubygems_hook.rb:134:in `chdir'
	25: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/rubygems_hook.rb:137:in `block in document'
	24: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/generator/ri.rb:27:in `generate'
	23: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/store.rb:769:in `save'
	22: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/store.rb:769:in `each'
	21: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/store.rb:770:in `block in save'
	20: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/store.rb:915:in `save_page'
	19: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/store.rb:915:in `open'
	18: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/store.rb:916:in `block in save_page'
	17: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/store.rb:916:in `dump'
	16: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/top_level.rb:203:in `marshal_dump'
	15: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/text.rb:133:in `parse'
	14: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/comment.rb:203:in `parse'
	13: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/text.rb:139:in `parse'
	12: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/markdown.rb:621:in `parse'
	11: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/markdown.rb:790:in `parse'
	10: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/markdown.rb:385:in `parse'
	 9: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/markdown.rb:481:in `apply'
	 8: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/markdown.rb:855:in `_root'
	 7: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/markdown.rb:481:in `apply'
	 6: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/markdown.rb:877:in `_Doc'
	 5: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/markdown.rb:481:in `apply'
	 4: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/markdown.rb:926:in `_Block'
	 3: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/markdown.rb:481:in `apply'
	 2: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rdoc-6.3.1/lib/rdoc/markdown.rb:15955:in `_Table'
	 1: from /Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
/Users/satoshinamai/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require': cannot load such file -- rdoc/markup/table (LoadError)
```

</details>

**Comment by igoriuz** [comment]
Would be nice to release this update since it breaks a complete rbenv (ruby 3.0.1) setup on M1 macs when upgrading all gems via "gem update"

**Comment by aladh** [comment]
Any idea when the fix will be released?

**Comment by igoriuz** [comment]
I would appreciate a ETA for releasing this fix since we already wait nearly 2 months. This is the problem for https://github.com/ruby/psych/issues/505 which breaks complete ruby setup via rbenv on M1 macs. (Doesn't break whole ruby setup on intel based systems..)

Just try installing fastlane and afterwards "gem update" to update everything..

Maybe @hsbt can have a look at it?

**Comment by hsbt** [comment]
I released rdoc-6.3.2 now.
